docs/typing.md

```
@@ -0,0 +1,59 @@
+|---|---|
+| `src/utils/http_utils.py` | JTN-525 |
+| `src/utils/security_utils.py` | JTN-525 |
+
+`src/utils/http_cache.py` was deliberately deferred from the initial strict
+set because it was recently refactored (JTN-493) and is still settling.  It
+will be added in a follow-up once it stabilizes.
+
+## How to add a module to the strict subset
+
+1. **Run mypy strict on the module locally** and fix all errors:
+   ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)